java/test/jmri/SensorTest.java

```diff
@@ -0,0 +1,295 @@
+package jmri;
+
+import java.beans.PropertyChangeEvent;
+import java.beans.PropertyChangeListener;
+import java.beans.PropertyVetoException;
+import java.util.ArrayList;
+import java.util.Set;
+import junit.framework.Test;
+import junit.framework.TestCase;
+import junit.framework.TestSuite;
+import org.junit.Assert;
+
+/**
+ * Tests for the Light class
+ *
+ * @author	Bob Jacobsen Copyright (C) 2008, 2010
+ */
+public class SensorTest extends TestCase {
+
+    @SuppressWarnings("all")
+    public void testStateConstants() {
+        Assert.assertTrue("On and Off differ", (Sensor.ON & Sensor.OFF) == 0);
+        Assert.assertTrue("On and Unknown differ", (Sensor.ON & Sensor.UNKNOWN) == 0);
+        Assert.assertTrue("Off and Unknown differ", (Sensor.OFF & Sensor.UNKNOWN) == 0);
+        Assert.assertTrue("On and Inconsistent differ", (Sensor.ON & Sensor.INCONSISTENT) == 0);
+        Assert.assertTrue("Off and Inconsistent differ", (Sensor.OFF & Sensor.INCONSISTENT) == 0);
+    }
+
+    public void testSensor() throws JmriException {
```

Comment:
I think this test would be better as an addition to jmri.impelmentation.AbstractSensorTestBase.  That way it gets run against all of the actual sensor implementations and not the MySensor class you added here.

Reply:
I agree. Fixed.